erpnext/patches/v11_0/update_delivery_trip_status.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('stock', 'doctype', 'delivery_trip')
+
+	for trip in frappe.get_all("Delivery Trip"):
+		trip_doc = frappe.get_doc("Delivery Trip", trip.name)
+
+		status_map = ["Draft", "Scheduled", "Cancelled"]

```

Comment:
I let out an audible "wat" cause it took me a while to understand this, I know this **works** but this needs to be a dict.

Reply:
But why though? It's just an index map anyway. :)

Reply:
Dunno, docstatus is a confusing thing as it is, code around it should at least be more clear

Reply:
Hm, fair enough. I'll make a change.